recipe-server/client/control/components/Fields.js

```diff
@@ -50,34 +50,43 @@ buildControlField.propTypes = {
   children: pt.node,
 };
 
+export const CheckboxGroup = ({ name, onChange, options = [], value }) => {
+  /**
+   * Checkbox change event handler. Appends or removes the selected checkbox's
+   * value to the existing `value` prop, and reports the change up to redux-form.
+   *
+   * @param  {Event} onChange event object
+   */
+  const handleChange = ({ event: target }) => {
+    if (target.checked) {
+      // Use Set to remove any dupes from the new value array
+      const newValue = new Set(value.concat([target.value]));
```

Comment:
This specific bit has been replaced with `_.uniq`, which uses arrays, but if we _were_ to use Sets, I think that'd be worth opening a separate PR for. I'm not sure how `fetch` handles using Set as a query param, but we'd probably have to do some sort of array-ifing when interacting with the API. I will look into it.